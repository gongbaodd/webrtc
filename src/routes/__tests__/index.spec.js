import { render } from "@testing-library/svelte";
import Index from "../index.svelte";

describe("index component", () => {
  test("should render component correctly", () => {
    const { container } = render(Index);

    expect(container).toMatchSnapshot();
  });
});
