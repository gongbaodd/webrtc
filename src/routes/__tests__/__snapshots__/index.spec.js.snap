// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index component should render component correctly 1`] = `
<body>
  <div />
</body>
`;
